src/benchmarks/real-world/ImageSharp/Images/Input/Tiff/Benchmarks/gen_big.ps1

```diff
@@ -0,0 +1,14 @@
+$Gm_Exe = "C:\Program Files\ImageMagick-7.0.10-Q16-HDRI\magick.exe"
```

Comment:
I'm not sure about tiff benchmarks, but I doubt you want to run benchmarks on CI which need files generated by ImageMagick :)